api/app/services/simplified_rag_service.py

```
@@ -1240,6 +1270,33 @@ async def query(
 
+            has_canonical_fix = any(
+                doc.metadata.get("_canonical_fix_injected") is True for doc in docs
+            )
+            noncanonical_docs = [
+                doc
+                for doc in docs
+                if doc.metadata.get("_canonical_fix_injected") is not True
+            ]
+            noncanonical_scores = [
+                score
+                for doc, score in zip(docs, doc_scores)
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)